inst/tutorials/introduction/introduction.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+title: "malariasimulation: an introductory tutorial"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+library(learnr)
+library(malariasimulation)
+library(dplyr)
+library(ggplot2)
+library(tidyr)
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+# `malariasimulation` overview
+
+`malariasimulation` in an individual based malaria model. This tutorial is an entry-level guide to using the model package in R.
+
+Use of `malariasimulation` can be divided into four stages:
+
+
+![](images/malsim_overview.png){width=100%}
+
+We will look at each of these in turn.
+
+
+## 1: Generate parameter set
+
+The key function here is `get_parameters()`
+
+Try running it to see what it generates.
+
+```{r get_parameters, exercise=TRUE}
+get_parameters()
+```
+
+You should see a long list of parameters - it's a complicated model!
+
+To understand what each parameter does in the model, you can go to the [help page](https://mrc-ide.github.io/malariasimulation/reference/get_parameters.html) (or use `?get_parameters()` when you are in R to see the same information).
+
+
+Please don't try to understand all of these right now! Instead, use the help page to try answering the questions below for now, and if there are any parameters you'd like to know more about, feel free to ask a demonstrator.
+
+
+```{r quiz}
```

Comment:
Might be worth adding some hints for these?
https://pkgs.rstudio.com/learnr/articles/exercises.html#r-code-hints

Reply:
I'm not sure you can add a hint to a question - it looks like these are only for code chunks.